rpc/documentation/api.md

```diff
@@ -1615,6 +1681,53 @@ wallet's relevant transactions contained therein.
 
 ___
 
+#### `TicketDetails`
+
+The `TicketDetails` message is included in responses to report transactions
+relevant to the wallet.  The message includes details such as which previous
+wallet inputs are spent by this transaction, whether each output is controlled
+by the wallet or not, the total fee (if calculable), and the earlist time the
+transaction was seen.
+
+- `bytes hash`: The hash of the given ticket.
+
+- `bytes spender_hash`: The hash of the spender (if applicable, otherwise empty).
+
+- `int64 ticket_age`: If unspent the blocks since being mined.  Otherwise,
+   the number of blocks from when it was mined to when it was spent.
```

Comment:
This is confusing to me and I'm not sure how it would be useful, even more so since it will take on different values depending on if it was spent.